unime/src/i18n/en/index.ts

```diff
@@ -189,26 +191,60 @@ const en = {
       DATA: 'Data',
       BADGES: 'Badges',
     },
+    CREDENTIAL: {
```

Comment:
I think `CREDENTIAL` should be on the top level (not under `ME`), since it can be accessed from anywhere throughout the app (similar to `BADGE`).